expfmt/openmetrics_create_test.go

```diff
@@ -397,14 +403,16 @@ request_duration_microseconds_count 2693
 				Metric: []*dto.Metric{
 					&dto.Metric{
 						Counter: &dto.Counter{
-							Value: proto.Float64(42),
+							Value:            proto.Float64(42),
+							CreatedTimestamp: openMetricsTimestamp,
 						},
 					},
 				},
 			},
 			out: `# HELP foos Number of foos.
 # TYPE foos counter
 foos_total 42.0
+foos_created 12345.6
```

Comment:
Are we sure about the order?

Reply:
Yes, this is the correct order.

https://github.com/OpenObservability/OpenMetrics/blob/main/specification/OpenMetrics.md#metricfamily-1